prediction_market_agent/agents/utils.py

```diff
@@ -18,16 +17,22 @@
 class AgentIdentifier(str, Enum):
     THINK_THOROUGHLY = "think-thoroughly-agent"
     MICROCHAIN_AGENT_OMEN = "microchain-agent-deployment-omen"
+    MICROCHAIN_AGENT_OMEN_TEST = "microchain-agent-deployment-omen_test"
+    MICROCHAIN_AGENT_OMEN_LEARNING_0 = "general-agent-0"
```

Comment:
Named them just `general-agent-X`, because it was too long for the select box 😄

Reply:
ahh fair enough 😄